packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -149,4 +160,5 @@ export interface Lb3AppBooterOptions {
 interface Lb3Application extends ExpressApplication {
   handler(name: 'rest'): ExpressRequestHandler;
   dataSources?: {[name: string]: unknown};
+  models?: {[name: string]: unknown};
```

Comment:
The actual type is `typeof juggler.PersistedModel`, but unfortunately we don't have access to that type in `@loopback/booter-lb3app`. 

Few ideas to consider:

(1)
Add typings to `loopback` package. To keep it simple, we can start by re-exporting loopback-datasource-juggler types. Maybe move `Lb3Application` to `loopback` along the way. This will give us strong type checks for code like `lb3app.models.User.create`.

```suggestion
  models?: {[name: string]: typeof juggler.PersistedModel};
```

Unfortunately, this will not help us with custom model methods like `User.login`. 

There is also a bug in juggler typings which prevents code like `lb3app.models.User.create({email: 'test@example.com'})` to compile, because PersistedModel does not have any properties and thus object `{email}` cannot be assigned to interface `ModelData<PersistedModel>`.

Considering those issues, I think we should not pursue this approach.

(2)
Considering that LB3 is pure JavaScript with no types, maybe we can use `any` type here? It will effectively disable compiler checks for any code accessing LB3 models methods, but I think that's ok considering that there were no type checks in the original LB3 application code base either.

```suggestion
  models?: {[name: string]: any};
```

WDYT?

Reply:
I like the second approach 👍